### PR TITLE
[closes #359] Add quote deletion logging

### LINKED ISSUE #359: Quote deletion logging
Including who voted

### PR DESCRIPTION
**Pull request by H4ckerJ4cker**
When a quote gets deleted the bot also sends a log message to the logging channel which includes the original quote and a list of the users the quote was deleted by.

### CODE REVIEW

**Comment by Skir0s** [comment]
I'm going to suggest not merging this PR until #382 is merged, just to avoid any conflicts.

File: cdbot/cogs/fun.py [hunk 1]
```diff
@@ -256,7 +258,20 @@ async def on_raw_reaction_add(self, raw_reaction: RawReactionActionEvent):
                     await connection.execute(
                         "DELETE FROM quotes WHERE quote_id = $1", reaction.message.id
                     )
+                reaction_users = set()
+                async for user in reaction.users():
+                    reaction_users.add(user)
```
Comment (AnonGuy) [inline cdbot/cogs/fun.py:263]: Why?

Comment (H4ckerJ4cker) [reply cdbot/cogs/fun.py:263]: It errors if you don't use async, and I tried it in a few ways, if there is a simpler way to achieve the same result please lmk.

File: cdbot/cogs/fun.py [hunk 2]
```diff
@@ -256,7 +258,20 @@ async def on_raw_reaction_add(self, raw_reaction: RawReactionActionEvent):
...
+                for quote_embed in reaction.message.embeds:
+                    embed = Embed(
+                        color=Colour.blue(),
+                        title="Quote Deleted",
+                        description=quote_embed.description
+                    )
+                    embed.add_field(name="Deleted By", value=f"{', '.join(user.mention for user in reaction_users)}")
```
Comment (AnonGuy) [inline cdbot/cogs/fun.py:270]:
```suggestion
                mentions = ', '.join(user.mention async for user in reaction.users())
                for quote_embed in reaction.message.embeds:
                    embed = Embed(
                        color=Colour.blue(),
                        title="Quote Deleted",
                        description=quote_embed.description
                    )
                    embed.add_field(name="Deleted By", value=mentions)
```

Comment (H4ckerJ4cker) [reply cdbot/cogs/fun.py]: In my testing your throw a TypeError.

**Pull request merged**